src/lib/core/overlay/overlay-directives.ts

```diff
@@ -103,14 +105,19 @@ export class ConnectedOverlayDirective implements OnDestroy {
   constructor(
       private _overlay: Overlay,
       templateRef: TemplateRef<any>,
-      viewContainerRef: ViewContainerRef) {
+      viewContainerRef: ViewContainerRef,
+      @Optional() private _dir: Dir) {
     this._templatePortal = new TemplatePortal(templateRef, viewContainerRef);
   }
 
   get overlayRef(): OverlayRef {
     return this._overlayRef;
   }
 
+  get dir(): 'ltr' | 'rtl' {
```

Comment:
Ah, didn't know about that type. Handy.  I think I still need it in case someone isn't using `dir`